Sources/Orbit/Support/Environment Keys/BackgroundColorKey.swift

```diff
@@ -0,0 +1,68 @@
+import SwiftUI
+
+/// Stores background default and active color used for Orbit components that contain background, such as `Card` or `Tile`.
+public struct BackgroundColor {
+    
+    public let inactive: any ShapeStyle
+    public let active: any ShapeStyle
+    
+    @ViewBuilder public var inactiveView: some View {
+        switch inactive {
+            case let color as Color:            color
+            case let color as LinearGradient:   color
+            case let color as RadialGradient:   color
+            default:                            EmptyView()
+        }
+    }
+    
+    @ViewBuilder public var activeView: some View {
+        switch active {
+            case let color as Color:            color
+            case let color as LinearGradient:   color
+            case let color as RadialGradient:   color
+            default:                            EmptyView()
+        }
+    }
+}
+
+struct BackgroundColorKey: EnvironmentKey {
+    static var defaultValue: BackgroundColor?
```

Comment:
`let`

Reply:
This is a shorthand to not having to specify `= nil`

Reply:
✔️